test.py

```diff
@@ -160,8 +160,16 @@ def spreadTest(generator, expected):
         # sort for easy comparison
         actual[j] = ''.join(sorted(actual[j]))
 
-    assert arraysEqual(expected, actual), "In a large sample of generated slugids, the range of characters found per character position in the sample did not match expected results.\n\nExpected: " + str(expected) + "\n\nActual: " + str(actual)
+    arraysEqual(expected, actual), "In a large sample of generated slugids, the range of characters found per character position in the sample did not match expected results.\n\nExpected: " + str(expected) + "\n\nActual: " + str(actual)
 
 def arraysEqual(a, b):
     """ returns True if arrays a and b are equal"""
```

Comment:
Previously this was returning a boolean, but now the assert is inside - I prefer the method checking for equality rather than performing an assert inside.